Casks/instatus-out.rb

```diff
@@ -0,0 +1,23 @@
+cask "instatus-out" do
+  version "1.0.8"
+  sha256 "14ad0e0e13dd3b5747defccfd2483069297fedcd0d74727246d031bd23ab5649"
+
+  url "https://github.com/instatushq/out/releases/download/v#{version}/Instatus.Out-#{version}-mac.dmg",
+      verified: "github.com/instatushq/out"
+  name "instatus-out"
```

Comment:
```suggestion
  name "Instatus Out"
```

This is the name of the app, not a repetition of the token.